client.cpp

```diff
@@ -423,7 +423,12 @@ void client::handle_event(short evtype)
         new_evtype = EV_READ;
     }
     if (evbuffer_get_length(m_write_buf) > 0) {
-        assert(finished() == false);
+        if (finished()) {
+            if (m_config->requests > 0 )
+                assert(false && "finished to write all requests but write buffer is not empty");
```

Comment:
If `m_config->requests > 0` doesn't it mean that there *are* more requests to write? Shouldn't it be `if (m_config->requests == 0`?

Reply:
@dori-lb m_config->requests states the requests threshold given to memtier, m_reqs_processed is the request counter itself.